terragraph_planner/optimization/topology_interference.py

```diff
@@ -319,15 +326,25 @@ def _calculate_net_gain_on_rx_links(
 
             # Calculate net gain using the angle between the links
             tx_dev = angle_delta(
-                none_throws(tx_interfering_link.tx_dev),
                 none_throws(interfering_path.tx_dev),
+                none_throws(tx_interfering_link.tx_dev),
```

Comment:
@pohanhf please make sure you agree with this.

Reply:
I think it is correct.